lisa/sut_orchestrator/azure/common.py

```diff
@@ -192,3 +194,18 @@ def get_environment_context(environment: Environment) -> EnvironmentContext:
 def wait_operation(operation: Any) -> Any:
     # to support timeout in future
     return operation.wait()
+
+
+def get_or_create_storage_container(
```

Comment:
@squirrelsc, Remind if we allow this kind of function name, 'get_or_create'? We followed @lizzha guideline, but what do you do in LISAv3?

Reply:
I think Liz's guideline is for Powershell code. The Python should be different. get_or_create is used a lot in Azure's python sdk.